design-docs/handling-encodings.md

```diff
@@ -0,0 +1,87 @@
+# Encoding
+
+Updated: __2016/03/19__
+Status: __Under Review__
+
+This document describes the rules and conventions for handling the character encoding of file contents in general, and file names 
+and comments in archives. It applies to tasks which need to read, write and transform text files, as well as tasks which create
+archives (zip files, tar files, etc.)
+
+## Use Cases
+
+### File Content Encoding
+
+Any time we need to generate a text file, or to read, transform and write the contents of a text file, we need to know what 
+character encoding(s) to use.
+
+Common situations include:
+
+* Copying a file from one location to another.
+* Creating an archive (zip, tar, etc.).
```

Comment:
No, they are binary formats. They contain filenames and comments in the archive metadata, which is where the coding comes into play, but the format is binary. And the spec only supports cp437 (basically ascii for visible characters), or utf8 if a special bit is set. See https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT appendix D.1 and D.2.